thinc/tests/test_util.py

```diff
@@ -46,7 +46,7 @@ def test_array_module_cpu_gpu_helpers():
     assert not is_numpy_array((1, 2))
 
 
-@given(label_smoothing=strategies.floats(min_value=0.0, max_value=0.5))
+@given(label_smoothing=strategies.floats(min_value=0.0, max_value=0.49))
```

Comment:
I think it's better to keep it at 0.5 and use `exclude_max`:

https://hypothesis.readthedocs.io/en/latest/data.html?highlight=floats#hypothesis.strategies.floats